slipcover/bytecode.py

```diff
@@ -572,7 +607,7 @@ def replace_inserted_function(self, offset, new_func_index):
 
         assert op == op_LOAD_CONST
 
-        # FIXME use actual argument lenth rather than art_ext_needed
+        # FIXME use actual argument length rather than arg_ext_needed
```

Comment:
Can you explain why this needs to be FIXed?

Reply:
We want our replacement code to have the same length, so to use the same number of `EXTENDED_ARG` opcodes; but `arg_ext_needed` indicate the minimum needed to encode an argument, not how many there actually were (and we're replacing). I'll improve the comment.

Reply:
Usually they are the same, but it is _possible_, if not likely, that unnecessary `EXTENDED_ARG`s were used.